docs-new/docs/graphql/core/actions/action-handlers.mdx

```diff
@@ -286,43 +262,28 @@ X-Hasura-Role: admin
   }
 }
 ```
+</TabItem>
+</Tabs>
 
-<div class="note">
-
-<div class="title">
+:::info Note
 
-Note
+Before creating an action via the [create_action metadata API](/graphql/core/api-reference/metadata-api/actions.mdx), all custom types
+need to be defined via the [metadata_set_custom_types](/graphql/core/api-reference/metadata-api/custom-types.mdx) metadata API.
```

Comment:
label is only `set_custom_types` for link - the `metadata_` is for id only - please refer original file for label and id vorrectness.

id => `.. _metadata_set_custom_types`
heading label => `set_custom_types`